cognite/neat/graph/stores/_base.py

```diff
@@ -28,7 +23,7 @@
 prom_sq = Gauge("store_single_query_time", "Time spent processing a single query", ["query"])
 
```

Comment:
See below implementation

```suggestion
if sys.version_info >= (3, 11):
    from enum import StrEnum
    from typing import Self
else:
    from backports.strenum import StrEnum
    from typing_extensions import Self


class MIMETypes(StrEnum):
    xml = "application/rdf+xml"
    turtle = "text/turtle"
    n_triples = "application/n-triples"
    n_quads = "application/n-quads"
    trig = "application/trig"
```